Marlin/language_tr.h

```diff
@@ -30,213 +30,214 @@
 #ifndef LANGUAGE_TR_H
 #define LANGUAGE_TR_H
 
+//#define SIMULATE_ROMFONT

```

Comment:
Hmm… Actually not all the languages I listed have two language files. Some of them only have one. Are they actually UTF-8 in those cases, without any non-UTF-8 versions?